lib/store/inventory.ex

```diff
@@ -125,8 +129,19 @@ defmodule Store.Inventory do
       [%Order{}, ...]
 
   """
-  def list_orders do
-    Repo.all(Order)
+  def list_orders(params \\ %{})
+
+  def list_orders(_opts) do
+    Order
+    |> apply_filters(_opts)
+    |> Repo.all()
+  end
+
+  alias Store.Inventory.Orders_Products
+
+  def list_order_products(params \\ %{}) do
+    Orders_Products
+    |> Repo.all()
   end
```

Comment:
This is listing all the entities from the `order_products` table. I think it would be nice to have a function that receives an `order_id` and lists all the products from that order.